tests/yast2_gui/yast2_instserver.pm

```diff
@@ -122,7 +127,13 @@ sub test_http_instserver {
     send_key_until_needlematch("yast2-instserver_sr0dev", "down", 4);
     send_key_and_wait("alt-n", 2);
     send_key_and_wait("alt-o", 2);
-    send_key_until_needlematch([qw(yast2-instserver-ui yast2-instserver-change-media)], 'alt-f10', 21, 30);
+    if (is_sle('=15-SP4')) {
```

Comment:
why we add this new one?

Reply:
This line added in the PR for the workaround also, so I think need to set condition for this line also.

Reply:
I got confused, because actually this is the right way to add the workaround, with the `else` part to not need to go the commit to revert it. well done.